scripts/ubuntu-bartender/ubuntu-bartender

```diff
@@ -97,17 +97,17 @@ fi
 # Please preface each provider default with the provider name
 
 # AWS SPECIFIC DEFAULTS
-AWS_AMI_NAME_FILTER="ubuntu/images-testing/hvm-ssd/ubuntu-bionic-daily-amd64-server-*"
 AWS_PROFILE="default"
 AWS_INSTANCE_TYPE="m5.large"
 # Official Canonical OwnerId
 AWS_AMI_OWNER=099720109477
 AWS_KEYPAIR_NAME=$USER
 AWS_ARM_BUILD=false
 
+
 # MULTIPASS SPECIFIC DEFAULTS
 MULTIPASS_DISK_SIZE=50G
-MULTIPASS_IMAGE=daily:b # daily image of bionic
+MULTIPASS_IMAGE=daily:f # daily image of bionic
```

Comment:
the comment is still wrong (but I would prefer to just replace `f` with the full name (`focal`) and drop the comment